vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -132,6 +139,57 @@ public String toString()
         });
     }
 
+    /**
+     * Waits for an <b><i>element</i></b> with the specified <b>locator</b> to stop moving
+     * @param locator locator to locate element
+     */
+    @When("I wait until element located by `$locator` stops moving")
+    public void waitForElementStopsMoving(Locator locator)
+    {
+        uiContext.getOptionalSearchContext().ifPresent(context -> {
+            locator.getSearchParameters().setWaitForElement(false);
+            IExpectedSearchContextCondition<Boolean> condition = new IExpectedSearchContextCondition<>() {
+                private Point lastPosition;
+                private boolean isElementPresent = true;
+
+                @Override
+                public Boolean apply(SearchContext context)
+                {
+                    try
+                    {
+                        return searchActions.findElement(context, locator).map(WebElement::getLocation)
```

Comment:
do we really need to find the element each time?

Reply:
changed, thanks